lib/actuators/include/Servo.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "Actuator.h"
```

Comment:
I did some googling and it looks like forward declaring classes doesn't work if the incomplete class is a base class, it  doesn't compile. http://umich.edu/~eecs381/handouts/IncompleteDeclarations.pdf

Reply:
I'd also be careful with this. "Note: forward declaration does prevent more inlining and optimizations. It's recommended to use Link Time Optimization or Link Time Code Generation for release builds."

We also don't use LTO rn. It can be added (https://stackoverflow.com/questions/31355692/how-do-i-enable-link-time-optimization-lto-with-cmake) but i would be hesitant because some things may break with LTO on micros (https://stackoverflow.com/questions/50175117/what-gets-discarded-by-gccs-flto). Maybe something to do when we have better access to all the hardware so we can debug those issues more easily.